### PR TITLE
docs: add changelog

### LINKED ISSUE #169: Add a changelog system
pypa uses https://pypi.org/project/towncrier mostly, though we can also check https://github.com/nedbat/scriv as an option.

### PR DESCRIPTION
**Pull request by FFY00**
Fixes #169

Signed-off-by: Filipe Laíns <lains@riseup.net>

### CODE REVIEW

**Review by gaborbernat — APPROVED** [review]
How do we generate this? Is it manually updated? 🤔

File: docs/changelog.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+../CHANGELOG.rst
```
Comment (uranusjr) [inline docs/changelog.rst:1]: Is this correct? I thought a `.. include:` derivative is needed.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
++++++++++
+Changelog
++++++++++
+
+
+Unreleased
+==========
+
+- Check dependencies recursively (`PR #183`_, Fixes `#25`_)
+- Build wheel and sdist distributions in separate envrionments, as they may have different dependencies (`PR #195`_, Fixes `#194`_)
```
Comment (layday) [inline CHANGELOG.rst:10]: envrionments -> environments

File: docs/changelog.rst (hunk 1 above)
Comment (FFY00) [reply docs/changelog.rst:1]: Github doesn't show file flags, but that is a symlink.

**Comment by FFY00** [comment]
> How do we generate this? Is it manually updated? thinking
> 

Yes, manually updated.

**Comment by gaborbernat** [comment]
> Yes, manually updated.

Would prefer something automatic, but we can start with this and add stuff on top of it.

File: docs/changelog.rst (hunk 1 above)
Comment (uranusjr) [reply docs/changelog.rst:1]: Oh, makes sense, thanks.

**Pull request merged**